src/kubectlUtils.ts

```diff
@@ -339,19 +341,20 @@ export async function getResourceAsJson<T extends KubernetesResource | Kubernete
  * Parse column based output which is seperated by whitespace(s) from kubectl or similar sources
  * for example, kubectl get po
  * @param lineOutput raw output with headers from kubectl or similar sources
+ * @param parsingRegex regex to parse the columns

```

Comment:
I would say something like "a regex for the column separators" - `parsing` doesn't tell you *what* it parses.

Reply:
Added new description